neo_batterylevelshutdown/buttons.py

```diff
@@ -0,0 +1,261 @@
+
+        :param channel: The pin number that has been pressed and thus is registering a 0
+        :return: time original button pressed, time both buttons were pressed
+        '''
+
+        # otherChannel is the button that has not been passed in by the channel parameter.
+        otherChannel = self.USABLE_BUTTONS[0] if channel == self.USABLE_BUTTONS[1] else \
+            self.USABLE_BUTTONS[1]
+
+        # there are two timers here.  One is for total time the original button was pushed.
+        # The second is for when the second button was pushed.  The timer gets restarted if
+        # the button is not pressed or is released.  The reason for the recorder is that if
+        # it is not kept, then when you let off the second button it will bounce and give a
+        # false reading.  Here we keep the highest consecutive time it was pushed.
+        startTime = time.time()     # time original button is pushed
+        dualStartTime = time.time() # time both buttons were pushed.
+        dualTimeRecorded = 0        # to prevent time being reset when letting off of buttons
+
+        while GPIO.input(channel) == 0:         # While original button is being pressed
+            if GPIO.input(otherChannel) == 1:   # move start time up if not pressing other button
+                dualButtonTime = time.time() - dualStartTime # How long were both buttons down?
+                if dualButtonTime > dualTimeRecorded:
+                    dualTimeRecorded = dualButtonTime
+                dualStartTime = time.time()     # reset start time to now
+
+        buttonTime = time.time() - startTime    # How long was the original button down?
+        return buttonTime, dualTimeRecorded
+
+    def chooseCancel(self):
+        """ method for use when cancelling a choice"""
+        logging.debug("Choice cancelled")
+        self.command_to_reference = ''  # really don't want to leave this one loaded
+        self.display.switchPages()      # drops back to the admin pages
+        # reset the display power off time
+        self.hat.displayPowerOffTime = time.time() + self.hat.DISPLAY_TIMEOUT_SECS
+
+    def chooseEnter(self, pageStack):
+        """ method for use when enter selected"""
+        logging.debug("Enter pressed.")
+        if pageStack == 'admin':
+            if self.display.checkIfLastPage():  # if true, go back to admin pageStack
+                self.display.switchPages()      # swap to status pages
+            else:
+                # find page name before we change it
+                self.command_to_reference = self.display.getAdminPageName()
+                logging.debug("Leaving admin page: {}".format(self.command_to_reference))
+                logging.debug("Confirmed Page shown")
+                self.display.showConfirmPage()
+        else:
+            logging.debug("Choice confirmed")
+            self.display.showWaitPage()
+            self.display.pageStack == 'wait'
```

Comment:
```suggestion
            self.display.pageStack = 'wait'
```

(pylint picked this up) - it may well be the cause of Dirk's display not blanking out.;